src/itest/java/com/iexec/sms/secret/compute/TeeTaskComputeSecretIntegrationTests.java

```diff
@@ -46,7 +47,7 @@
 public class TeeTaskComputeSecretIntegrationTests extends CommonTestSetup {
     private static final String APP_ADDRESS = "0xabcd1339ec7e762e639f4887e2bfe5ee8023e23e";
     private static final String UPPER_CASE_APP_ADDRESS = "0xABCD1339EC7E762E639F4887E2BFE5EE8023E23E";
-    private static final String SECRET_VALUE = "secretValue";
+    private static final String SECRET_VALUE = RandomStringUtils.randomAscii(4096);
```

Comment:
Are you sure about introducing randomness in tests? This may lead to hardly reproducible tests.

Reply:
Looks great!